source/game/field/ObjectCollisionConvexHull.cc

```diff
@@ -20,6 +22,27 @@ ObjectCollisionConvexHull::~ObjectCollisionConvexHull() {
     delete[] m_worldPoints.data();
 }
 
+/// @addr{0x807F957C}
+f32 ObjectCollisionConvexHull::getBoundingRadius() const {
+    return m_worldRadius;
+}
+
+/// @addr{0x80836628}
+const EGG::Vector3f &ObjectCollisionConvexHull::getSupport(const EGG::Vector3f &v) const {
+    const EGG::Vector3f *result = &v;
+    f32 minDot = v.dot();
+
+    for (auto &iter : m_worldPoints) {
+        f32 iterDot = iter.dot();
+        if (iterDot < minDot) {
+            result = &iter;
+            minDot = iterDot;
+        }
+    }
+
+    return *result;
+}
```

Comment:
```suggestion
const EGG::Vector3f &ObjectCollisionConvexHull::getSupport(const EGG::Vector3f &v) const {
    const EGG::Vector3f *result = &m_worldPoints[0];
    f32 minDot = v.dot(*result);

    for (size_t i = 1; i < m_worldPoints.size(); ++i) {
        const auto &iter = m_worldPoints[i];
        f32 iterDot = v.dot(iter);
        if (iterDot < minDot) {
            result = &iter;
            minDot = iterDot;
        }
    }

    return *result;
}
```
`iterDot < minDot` could be checked with the fcmpo instruction.